tools/tikv-control.md

```diff
@@ -0,0 +1,164 @@
+---
+title: TiKV Control User Guide
+category: tools
+---
+
+# TiKV Control User Guide
+
+TiKV Control (`tikv-ctl`) is a command line tool of TiKV, used to manage the cluster. When you compile TiKV, the `tikv-ctl` command is also compiled at the same time. If the cluster is deployed using Ansible, the binary file also exist in the corresponding `tidb-ansible/resources/bin` directory.
+
+## General options
+
+`tikv-ctl` provides two operation modes:
+
+- Remote mode: use the `--host` option to accept the service address of TiKV as the argument
+
+    For this mode, if SSL is started in TiKV, `tikv-ctl` also needs to specify the related certificate file. For example:
```

Comment:
Should be `SSL is enabled in TiKV`. The text in docs-cn is also wrong. I will fix the last one.